framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
The problem is that you've *states* on the one side and *enabled* flags on the other side. This shouldn't be mixed up by secondary sources.